### PR TITLE
Apply changes after refactoring in uhdm-integration repo

### PR DESCRIPTION
**Pull request by wtatarski**

### CODE REVIEW

**Comment by tgorochowik** [comment]
This needs to go in first, right: https://github.com/alainmarcel/uhdm-integration/pull/163 ?

**Comment by wtatarski** [comment]
> This needs to go in first, right: [alainmarcel/uhdm-integration#163](https://github.com/alainmarcel/uhdm-integration/pull/163) ?

Actually this should be merged as the last one -> https://github.com/alainmarcel/uhdm-integration/issues/177

File: sim/verilator-uhdm/meta.yaml
Comment (ajelinski) [inline sim/verilator-uhdm/meta.yaml:2]: Don't change this line. It doesn't influence the final version of the package as it is set by conda-build-prepare in CI.

File: syn/yosys-uhdm/meta.yaml
Comment (ajelinski) [inline syn/yosys-uhdm/meta.yaml:2]: Don't change this line. It doesn't influence the final version of the package as it is set by conda-build-prepare in CI.

**Review by ajelinski — CHANGES_REQUESTED** [review]
I can see that the CI jobs succeeded with each of the recipes containing a test command so it seems that the build script changes work.

The version lines need to be restored since changing them is pointless.

Otherwise, LGTM.

**Comment by wtatarski** [comment]
@ajelinski  
I resolved your issues, but there is a fault in `prjtrellis-windows` and `master-package`.  I did not change anything in those packages. So has it happened before?

**Comment by ajelinski** [comment]
I can see `prjtrellis-windows` failed because of QT servers again (there was a problem with their servers lately that lasted a couple of days):
```
   -- Downloading http://download.qt.io/official_releases/jom/jom_1_1_3.zip... Failed. Status: 28;"Timeout was reached"
```
Now this file is available so after rerunning tests the job has passed.

The PR changes LGTM now and can be merged when the related repositories got their PRs merged. I'll leave my review as "Changes requested" to not forget to change the `git_rev` before merging.

File: sim/verilator-uhdm/meta.yaml
Comment (ajelinski) [inline sim/verilator-uhdm/meta.yaml:9]: Change it before merging.

File: syn/yosys-uhdm/meta.yaml
Comment (ajelinski) [inline syn/yosys-uhdm/meta.yaml:9]: Change it before merging.

**Review by ajelinski — CHANGES_REQUESTED** [review]
LGTM. The review is still "Request changes" only to remember about changing branches in `git_rev`.

File: sim/verilator-uhdm/meta.yaml
Comment (wtatarski) [reply sim/verilator-uhdm/meta.yaml]: Changed to `uhdm-verilator`

File: syn/yosys-uhdm/meta.yaml
Comment (wtatarski) [reply syn/yosys-uhdm/meta.yaml]: Changed to `uhdm-yosys`

**Comment by wtatarski** [comment]
Changes in yosys and verilator was merged to base branches. Should be OK now.

**Comment by ajelinski** [comment]
I was wondering about the source repository changes but I can see these are the ones originally used before changes in `uhdm-integration`: https://github.com/alainmarcel/uhdm-integration/pull/163/files#diff-fe7afb5c9c916e521401d3fcfb4277d5071798c3baf83baf11d6071742823584L1

**Review by ajelinski — APPROVED** [review]

**Pull request merged**
